docs/release-notes/summer-2021/README.md

```diff
@@ -21,22 +21,14 @@
 | package name | version |
 |--------------|---------|
 | `@rei/cedar` | ^10.x.x |
+| `@rei/febs` | ^8.x.x |
 
+- Remove `@babel/preset-env` and `@babel/register` if they are included in your project.
+- If `@rei/browser-history` is included in your project, install `core-js@3.x.x`. Eventually `core-js` can be removed when `@rei/browser-history` has been updated to use `@rei/febs`.
```

Comment:
Is this worth mentioning in our release notes? It seems very specific to one or two projects.